go/tools/builders/compilepkg.go

```diff
@@ -343,6 +344,17 @@ func compileArchive(
 			cgoSrcs[i-len(goSrcs)] = coverSrc
 		}
 	}
+	if strings.Contains(outLinkObj, "external/") && slices.Contains(gcFlags, "-d=libfuzzer") {
```

Comment:
@rickystewart I couldn't find a cleaner way to determine whether we're compiling an _external_ dependency, but this seems to work. Maybe you have a better idea?

Reply:
This is probably how I would do it.